$NetBSD: patch-aa,v 1.1 2010/09/24 18:10:46 gdt Exp $

Don't install odd version file in a strange place.
Reported to tahoe development mailinglist.

--- setup.py.orig	2010-09-21 06:59:37.000000000 +0000
+++ setup.py
@@ -254,7 +254,7 @@ data_files = [(doc_loc, data_fnames)]
 # Note that due to a bug in distutils we also have to maintain a
 # MANIFEST.in file specifying cryptopp/extraversion.h. This bug was
 # fixed in Python 2.7
-data_files.append(('cryptopp', ['cryptopp/extraversion.h']))
+#data_files.append(('cryptopp', ['cryptopp/extraversion.h']))
 
 if ECDSA:
     long_description='RSA-PSS-SHA256 signatures, ECDSA(1363)/EMSA1(SHA-256) signatures, SHA-256 hashes, and AES-CTR encryption'
